src/main/java/seedu/address/logic/parser/DeleteCustomerOrderCommandParser.java

```diff
@@ -1,29 +1,33 @@
 package seedu.address.logic.parser;
 
-import seedu.address.logic.commands.AddCustomerOrderCommand;
 import seedu.address.logic.commands.DeleteCustomerOrderCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
-import java.util.ArrayList;
-
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
-
 public class DeleteCustomerOrderCommandParser implements Parser<DeleteCustomerOrderCommand> {
 
+    @Override
     public DeleteCustomerOrderCommand parse(String args) throws ParseException {
         requireNonNull(args);
 
-        String[] splitArgs = args.trim().split("\\s+");
```

Comment:
I think the original is better because it can split multiple whitespaces. Correct me if I am wrong!